src/server/internalServer.h

```diff
@@ -117,7 +117,7 @@ class InternalServer {
                                void** cont_cls);
     bool start();
     void stop();
-    std::string getAddress() { return m_addr; }
+    std::vector<std::string> getAddresses() { return m_addr; }
```

Comment:
Make `const` (and also please fix `const`-correctness of other getters in a separate commit)